cve_bin_tool/checkers/ncurses.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for Ncurses
+
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-72/product_id-38464/GNU-Ncurses.html
+
+RSS feed: http://www.cvedetails.com/vulnerability-feed.php?vendor_id=72&product_id=38464&version_id=0&orderby=3&cvssscoremin=0
+
+"""
+from ..util import regex_find
+
+
+def guess_contains_ncurses(lines):
+    """Tries to determine if a file includes Ncurses
+    """
+    for line in lines:
+        if "_nc_first_name" in line:
+            return 1
+    return 0
+
+
+def get_version(lines, filename):
+    """returns version information for Ncurses as found in a given file.
+    The version info is returned as a tuple:
+        [modulename, is_or_contains, version]
+
+    modulename will be ncurses if Ncurses is found (and blank otherwise)
+    is_or_contains idicates if the file is a copy of Ncurses or contains one
+    version gives the actual version number
+
+    VPkg: gnu, ncurses
+    """
+    regex = [r"ncurses ([0-9]+\.[0-9]+)"]
+    regex1 = [r"tic-([0-9]+\.[0-9]+)"]
```

Comment:
This signature makes me uncomfortable, beacuse it's a 3 letter differentiator and I feel like it's going to twig on something the ends in tic eventually.  Can you take a look at some other distros ncurses packages and see if that's the best we can do?

Reply:
@terriko  I added another regex, now version from regex1 and regex2 are used only if both of them are found. Possibly reducing the false positives